addons/website/static/src/snippets/s_process_steps/000.js

```diff
@@ -0,0 +1,156 @@
+/** @odoo-module */
+
+import { qweb } from 'web.core';
+import publicWidget from 'web.public.widget';
+
+const ProcessStepsWidget = publicWidget.Widget.extend({
+    selector: '.s_process_steps',
+    xmlDependencies: ['/website/static/src/snippets/s_process_steps/000.xml'],
+    disabledInEditableMode: false,
+    edit_events: {
+        'connector_changed': '_onConnectorChanged',
```

Comment:
I don't think this handler is ever called? This is the name of an Odoo event which you trigger_up, not a DOM event.